src/modules/shared/pages/innovation/export-requests/export-request-info.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { CoreComponent } from '@app/base';
+
+import { InnovationExportRequestStatusEnum } from '@modules/stores/innovation/innovation.enums';
+
+import { InnovationExportRequestInfoDTO } from '@modules/shared/services/innovations.dtos';
+import { InnovationsService } from '@modules/shared/services/innovations.service';
+
+
+@Component({
+  selector: 'shared-pages-innovation-export-request-info',
+  templateUrl: './export-request-info.component.html'
+})
+export class PageInnovationExportRequestInfoComponent extends CoreComponent implements OnInit {
+
+  innovationId: string;
+  requestId: string;
+  innovationRequest?: InnovationExportRequestInfoDTO;
+
+  // Flags
+  isInnovatorType: boolean;
+  isLeadershipTeamType: boolean;
```

Comment:
I personally prefer the `isSupportTeam` (stealing the name you gave in the IR page), thoughts?

Reply:
Solved.